### ISSUE TITLE
The cookie expiration time "" is not valid

### ISSUE THREAD

**Issue by patto-sk** [Codeception/Codeception#2301]
Codeception stopped working when testing payments via PayPal sandbox. Errors appear: [UnexpectedValueException] The cookie expiration time "" is not valid.

**Comment by Naktibalda** [comment]
Upgrade to 2.1.2, 
I fixed this issue already #2234

**Comment by patto-sk** [comment]
I'm using the latest version 2.1.2 but the error still persist

**Comment by Naktibalda** [comment]
Are you testing it with PhpBrowser? I will try to reproduce this issue later.

**Comment by patto-sk** [comment]
Yes, PhpBrowser.

**Comment by Naktibalda** [comment]
I failed to reproduce your issue.
Are you using [setCookie](http://codeception.com/docs/modules/PhpBrowser#setCookie) method in your test?

Please provide your test, complete output and PayPal url which is being tested.

**Comment by patto-sk** [comment]
No, I'm not using setCookie method. PayPal url is generated via token (Express Checkout). But I've resolved my problem by ignoring to test third-party service -> PayPal payments (all payments are auto-approved)

**Comment by cloudEdwards** [comment]
I am having the exact same problem with Paypal standard. Maybe there is some js magic that is happening with Paypal. If I recall Paypal sends a code back that we append to the end of ```www.sandbox.paypal.com/ca/cgi-bin/webscr```

**Comment by cloudEdwards** [comment]
I tested this using WebDriver browser tests and it works like a charm.

**Comment by Naktibalda** [comment]
You should report to Paypay that they generate invalid cookies.

**Comment by Naktibalda** [comment]
Could one of you give me a stack trace of this issue?

**Comment by cloudEdwards** [comment]
This is pretty weird. I tried to get a stack trace for you, but now it's working in acceptance. The only difference I can think of is that I was using git bash the first time, out side of my vagrant machine. But I cant copy the trace from gitBash, so I switched to Babun terminal. But Babun only works when I vagrant ssh into my VM, then I can run the tests. But when I ran the tests inside the VM, they worked.
